src/app/routes/api/v3/v3.routes.ts

```diff
@@ -1,11 +1,17 @@
 import { Router } from 'express'
 
 import { AdminRouter } from './admin'
+import { AnalyticsRouter } from './analytics'
 import { AuthRouter } from './auth'
+import { BillingRouter } from './billing'
+import { BounceRouter } from './bounce'
 import { UserRouter } from './user'
 
 export const V3Router = Router()
 
 V3Router.use('/admin', AdminRouter)
 V3Router.use('/user', UserRouter)
 V3Router.use('/auth', AuthRouter)
+V3Router.use('/notifications/bounce/email', BounceRouter)
```

Comment:
The router should be called `NotificationsRouter` since that is the router for the `/notifications` route.
prefer the root route to be `/notifications`, and pass the subroutes into `NotificationRouter`

Reply:
should we introduce a folder structure such as /notifications/bounce to reduce confusion between notifications and bounce? I feel that these two concepts should be decoupled following this logic

Reply:
sure thing